kafka_producer.js

```diff
@@ -191,13 +193,13 @@ function getKafkaPayload(req) {
 };
 
 // App Routes
-app.get('/', oidc.ensureAuthenticated(), agentForm.okta);
-app.get('/mental-health', oidc.ensureAuthenticated(), agentForm.mentalHealth);
-app.get('/authorization-code/callback', oidc.ensureAuthenticated(), agentForm.okta);
+// app.get('/', oidc.ensureAuthenticated(), agentForm.okta);
+// app.get('/mental-health', oidc.ensureAuthenticated(), agentForm.mentalHealth);
+// app.get('/authorization-code/callback', oidc.ensureAuthenticated(), agentForm.okta);
 
-app.get('/logout', oidc.forceLogoutAndRevoke(), agentForm.logout);
-app.post('/submit-ticket',  oidc.ensureAuthenticated(), agentForm.submitTicketCovid);
-app.post('/submit-ticket-mh',  oidc.ensureAuthenticated(), agentForm.submitMHTicket);
+// app.get('/logout', oidc.forceLogoutAndRevoke(), agentForm.logout);
+// app.post('/submit-ticket',  oidc.ensureAuthenticated(), agentForm.submitTicketCovid);
+// app.post('/submit-ticket-mh',  oidc.ensureAuthenticated(), agentForm.submitMHTicket);
```

Comment:
Revert this, and use the config param as mentioned above.

Reply:
Added a config to enable or disable OIDC